src/test/java/org/fundacionjala/coding/Movies/TestCustomer.java

```diff
@@ -0,0 +1,188 @@
+package org.fundacionjala.coding.Movies;
+
+import org.fundacionjala.coding.KevinHerrera.Movies.Customer.Customer;
+import org.fundacionjala.coding.KevinHerrera.Movies.Movie.Movie;
+import org.fundacionjala.coding.KevinHerrera.Movies.Movie.MovieFactory;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Kevin Herrera - AWT-[01].
+ * @version 0.1
+ */
+public class TestCustomer {
+    static final String REGULAR_MOVIE_TITLE = "Batman";
+    static final String NEW_RELEASE_MOVIE_TITLE = "Avengers";
+    static final String CHILDREN_MOVIE_TITLE = "Little Pig";
+
+    static final int DAYS_RENT_1 = 1;
+    static final int DAYS_RENT_2 = 2;
+    static final int DAYS_RENT_3 = 3;
+    static final int DAYS_RENT_5 = 5;
+    static final int DAYS_RENT_6 = 6;
+    static final int DAYS_RENT_10 = 10;
+
+    static final int RENT_AMOUNT_4 = 4;
+    static final int RENT_AMOUNT_10 = 10;
+    static final int RENT_AMOUNT_15 = 15;
+
+    static final int RENT_POINTS_2 = 2;
+    static final int RENT_POINTS_3 = 3;
+    static final int RENT_POINTS_4 = 4;
+    /**
+     * .
+     */
+    @Test
+    public void testCustomerName() {
+        Customer customerTest = new Customer("Juan");
+        assertEquals("Juan", customerTest.getName());
+    }
+
+    /**
+     * .
+     */
+    @Test
+    public void testCustomerRentalMovieBatmanFor2Days() {
+        Customer customerTest = new Customer("test");
+        Movie movie = MovieFactory.createRegularMovie(REGULAR_MOVIE_TITLE);
+        customerTest.rentMovie(movie, DAYS_RENT_2);
+        assertEquals("Batman", customerTest.getRentedMoviesList().get(0).getMovie().getTitle());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use already-defined constant 'REGULAR_MOVIE_TITLE' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)